src/patternfly/components/Banner/banner.scss

```diff
@@ -90,6 +90,11 @@
     --#{$banner}--Color: var(--#{$banner}--m-info--Color);
   }
 
+  &.pf-m-custom {
+    --#{$banner}--BackgroundColor: var(--#{$banner}--m-custom--BackgroundColor);
+    --#{$banner}--Color: var(--#{$banner}--m-custom--Color);
```

Comment:
I believe all these colors should be using the "on-" tokens - e.g. `--global-text-color-status-on-custom-default`

Reply:
I'm probably misunderstanding something, but I assign those tokens to the modifier vars [higher in the file](https://github.com/patternfly/patternfly/blob/12db7dae8e8c7a01be8be6d78484d46406a4c9e3/src/patternfly/components/Banner/banner.scss#L41). Do I need to redo something about how I did that?

Reply:
No I'm just an idiot today. Sorry, trying to do too many things at once.